pkg/devspace/kubectl/util.go

```diff
@@ -315,8 +316,7 @@ func GetPodStatus(pod *k8sv1.Pod) string {
 	if !initializing {
 		hasRunning := false
 
-		for i := len(pod.Status.ContainerStatuses) - 1; i >= 0; i-- {
-			container := pod.Status.ContainerStatuses[i]
+		for _, container := range pod.Status.ContainerStatuses {
```

Comment:
This changes the order the statuses are processed, is the order really not important?

Reply:
I would argue the order is very important here, because this could eventually lead to another reason being displayed

Reply:
Of course this is important. My bad. I reverted the change in a25f9fd7c7535f885b87e6e091c63b07b9c2c8e8